docs/src/pages/getting-started/installation/troubleshooting.react.mdx

```diff
@@ -90,3 +90,26 @@ export default defineConfig({
     "skipLibCheck": true,
 ...
 ```
+
+### Jest
+
+As of v2.15.0 of `@aws-amplify/ui-react` which included the release of Geo components, users of the Jest testing
+framework may run into the following error when attempting to run tests:
```

Comment:
Very minor nit: this newline should be deleted and let markdown handle newlines natively. Same with line 114.

Fyi, vs code has "Toggle word wrap" options you can use for local dev helps.